mmgen/models/architectures/biggan/generator_discriminator.py

```diff
@@ -274,6 +286,13 @@ def forward(self, noise, label=None, num_batches=0, return_noise=False):
             return_noise (bool, optional): If True, ``noise_batch`` and
                 ``label`` will be returned in a dict with ``fake_img``.
                 Defaults to False.
+            truncation (float, optional): Truncation factor. Give value not
+                less than 0., the truncation trick will be adopted.
+                Otherwise, the truncation trick will not be adopted.
+                Defaults to -1..
+            use_embedding (bool, optional): Whether to use `shared_embedding`
```

Comment:
Is `shared_embedding`accurate? In Biggan, there may be some model with split embedding?